src/config/ConfigLog.ts

```diff
@@ -17,6 +17,7 @@ const options = new LoggerFactoryOptions().addLogGroupRule(
 
 // Create a named loggerfactory and pass in the options and export the factory.
 // Named is since version 0.2.+ (it's recommended for future usage)
+// eslint-disable-next-line import/prefer-default-export
 export const factory = LFService.createNamedLoggerFactory(
```

Comment:
Why can't we use default export instead of disabling the rule?

Reply:
I didn't want to change the API (I know, I already did this with changing the ctype/txstatus/payload thingy)

Reply:
also the whole naming is weird here. I would rather move this logger to a util folder and use it from there

Reply:
https://github.com/KILTprotocol/sdk-js/issues/150